common/build-style/python3-pep517.sh

```diff
@@ -24,7 +24,10 @@ do_check() {
 		${make_install_args} ${make_install_target:-dist/*.whl}
 
 	PATH="${testdir}/usr/bin:${PATH}" PYTHONPATH="${testdir}/${py3_sitelib}" \
-		${make_check_pre} pytest3 ${testjobs} ${make_check_args} ${make_check_target}
+		${make_check_pre} pytest3 --ignore="${wrksrc}/tmp" ${testjobs} \
+		${make_check_args} ${make_check_target}
+
+	rm -r "${wrksrc}/tmp"
```

Comment:
Alternatively, we could do:
```suggestion
	rm -r "${testdir}"
	rmdir tmp >/dev/null 2>&1 || true
```

Reply:
I agree with removing only what was created in this run (i.e. just `${tmpdir}`).

Also, in terms of avoiding conflicts, maybe the tmpdir should be something more "unique". Perhaps `${wrksrc}/.xbps-tmpdir/$(data +%s)` which also has the advantage that pytest will not recurse hidden directories, so the `--ignore ..` doesn't seem to be necessary.

Reply:
I'm a bit confused about the need to remove the `$testdir` after a successful run. What's the issue if we keep it there?

And if we are adding `/.xbps-tmpdir` for uniqueness, I don't see why we also need to include another sub-directory under it. I only added the epoch subdirs for the extremely rare chance if a package actually ships a `/tmp` dirrctory under `$wrksrc`.

Reply:
Fine with me in principle. Although I'd be wary about what happens when there is an old testdir lying around from a previous check run. I don' t know if `python -m installer` will do the right thing.

Reply:
I guess this is irrelevant because the wheel was already built before the tmpdir was created, and the installer should really only be unpacking the wheel. I do prefer @tornaria's suggestion to give the directory a hidden, XBPS-specific name that is extremely unlikely to conflict with package contents. Maybe try changing `testdir` to `.xbps-testdir` (which is more meaningful than `.xbps-tempdir`) and see if that solves your problem without needing to ignore tests, and then skip the removal.

Reply:
After some testing, the python installer doesn't override files so moved the deletion logic at top. And removed the now redundant epoch subdir & ignore flag.